examples/hack/count.hh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/hhvm
+<?hh // strict
+
+use namespace HH\Lib\Str;
+use function HH\Lib\Experimental\IO\request_output;
+use function usleep;
+
+// Simple example script that counts to 10 at ~2Hz, then stops.
+
+<<__EntryPoint>>
+async function count_to_ten(): Awaitable<noreturn> {
+  $output = request_output();
+  for ($count = 1; $count <= 10; $count++) {
+    await $output->writeAsync(
+      Str\format("%d\n",$count)
+    );
+ 
+    // usleep is builtin, it is not an async builtin - so it also must block the main request thread
+    usleep(500000);
```

Comment:
HH\Asio\usleep()

Reply:
as a side note, async is not multithreading. Some async operations may use threading as an implementation detail, but in general (and for IO specifically), it's single-threaded.

Reply:
fixed here : #340